drivers/sensor/nrf5/Kconfig

```diff
@@ -9,6 +9,7 @@
 menuconfig TEMP_NRF5
 	bool "nRF5 Temperature Sensor"
 	depends on SOC_FAMILY_NRF
+	depends on SENSOR
```

Comment:
The reason I did this was because during my testing, I thought I only needed to set CONFIG_TEMP_NRF5=y in my prj.conf file to enable the TEMP_NRF5 sensor; as there is no reference from this temp_nrf5 Kconfig file to the overall SENSOR Kconfig file. But yes, I agree (Y)